modules/assim.batch/R/pda.utils.R

```diff
@@ -0,0 +1,671 @@
+##' Load Dataset for Paramater Data Assimilation
+##'
+##' @title Load Dataset for Paramater Data Assimilation
+##' @param input.settings = settings$assim.batch$inputs from pecan.xml or similar
+##'
+##' @return A list containg the loaded input data, plus metadata
+##'
+##' @author Ryan Kelly
+##' @export
+load.pda.data <- function(input.settings) {
```

Comment:
@ryankelly-uiuc @bcow lets get the load.pda.data function generalized and migrated out of assim.batch to be a more global util for loading files. Lets also separate the first nested set of if's, which really is about finding the file name (but will need to be generalized to also find the format record), from the process of actually loading the file (which will need to be generalized to look first for format-specific parsers and second for mime-type specific parsers).